models/aether-2.1.x/yang/onf-aether-types.yang

```diff
@@ -79,10 +79,10 @@ module onf-aether-types {
 
   typedef tac {
       type string {
-        length 0..8;
-        pattern '[0-9A-F\.]{0,8}';
+        length 0..6;
+        pattern '[0-9A-F\.]{0,6}';
```

Comment:
I think it would be better to make it {1,6} as zero length would cause problems I think. Same with others.